mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -141,59 +141,114 @@ def ensure_list(alist):  # {{{
     return alist  # }}}
 
 
-def get_datetimes(ds, timestr, yearoffset):  # {{{
+def parse_time(ds, inTimeVarName, inReferenceDate, outTimeVarName,
+               outReferenceDate):  # {{{
     """
-    Computes a list of datetimes from the time variable in the dataset ds with
-    variable name (or a list of 2 names) given by timestr, typically one of
-    'daysSinceStartOfSim', 'xtime', or ['xtime_start', 'xtime_end'].
+    Converts MPAS times to `netcdftime.DatetimeNoLeap` objects consistent with
+    MPAS's 365-day calendar.
```

Comment:
@milenaveneziani, could you clarify what you mean by "transient simulations"?  My understanding is that MPAS currently only supports a 365-day calendar but I could be wrong about that.

It shouldn't be that hard to specify the calendar as an input argument and have it default to `365_day`.  Until MPAS output is CF-compliant, it will be hard to know where to get that calendar from.  I guess it would need to be yet another config argument?